README.md

```diff
@@ -116,6 +123,41 @@ Tutorials hosted in other packages include (but haven't been recently vetted):
 * [How to make all-sky plots](https://github.com/desihub/desiutil/blob/master/doc/nb/SkyMapExamples.ipynb)
 * [Working with DESI target bits (Main, CMX, SV)](https://github.com/desihub/desitarget/blob/master/doc/nb/target-selection-bits-and-bitmasks.ipynb)
 
+# External databases
+
+In addition to the database access through NERSC, one can use the following external database services.
+
+## Astro Data Lab
+
+The Astro Data Lab serves a copy of the DESI EDR database as `desi_edr`. This is accessible to users without a NERSC account. 
+Various modes of data access are described [here](https://datalab.noirlab.edu/desi/access.php). 
+In particular, there is a Table Access Protocol (TAP) handle that provides a convenient access layer for the 
+DESI catalog database tables. TAP-aware clients (such as TOPCAT) can point to `https://datalab.noirlab.edu/tap`, 
+select the `desi_edr` database, and see the database tables and descriptions. 
+
+Descriptions of the associated tables can also be found in the Data Lab [table browser](https://datalab.noirlab.edu/query.php?name=desi_edr.zpix) 
+and on the DESI Data Documentation [Database page](https://data.desi.lbl.gov/doc/access/database/). 
+
+## SPARCL
+
+The [SPectra Analysis & Retrievable Catalog Lab (SPARCL)](https://astrosparcl.datalab.noirlab.edu) contains 
+DESI EDR spectra that were coadded per healpix. It is a searchable database that can be used via the Astro Data Lab 
+JupyterLab Notebook. Alternatively, the Python client can be installed locally using:
+```
+pip install sparclclient
+```
+The client can be loaded within a Python session or program via:
+```
+>> from sparcl.client import SparclClient 
+>> client = SparclClient()
+```
+
+There are instructions and useful examples in the [How-to-use-SPARCL tutorial notebook](https://github.com/astro-datalab/notebooks-latest/blob/master/04_HowTos/SPARCL/How_to_use_SPARCL.ipynb).
+
+# Acknowledgments
+
+Any use of DESI data whether via NERSC or external databases requires [DESI Data Acknowledgments](https://data.desi.lbl.gov/doc/acknowledgements/).
```

Comment:
Double-check the spelling: https://github.com/desihub/desidatadocs/blob/edr/docs/acknowledgments.md

Reply:
for context, there is a British vs. American spelling difference for "acknowledgments", and even the pubboard acknowledgments wiki page used both on a single page.  Current main uses British acknowledgements (with an extra "e") but the edr branch soon to be merged will switch to the American spelling "acknowledgments" as the final URL to use.  We need to check this on the NOIRLab websites too!

Reply:
Yes, we also had (have?) both versions of the spelling on the Data Lab website. After figuring out which is the American spelling, we have so far fixed it on the SPARCL website (https://astrosparcl.datalab.noirlab.edu/sparc/acknowledgments/).

That said, we won't be able to change the SPARCL website for a while (complicated -- it's tied to code releases) so are you able to add a re-direct such that "https://data.desi.lbl.gov/doc/acknowledgements/" re-directs to "https://data.desi.lbl.gov/doc/acknowledgments/" meaning that either spelling would not cause an error? (if not, a hack would be to create two identical "acknowledgments.md" and "acknowledgements.md" pages for now, which is of questionable taste but might work in a pinch).

Reply:
We have used the Acknowledgments in many places so tracking it all down might take a while even on the Data Lab website (which we can still modify today, unlike the SPARCL website).

Reply:
@sbailey  could you confirm that I should change the URL in the README (and everywhere we still possibly can) to "https://data.desi.lbl.gov/doc/acknowledgments/" without the British "e"?

Reply:
yes, please change it everywhere you can to the American spelling https://data.desi.lbl.gov/doc/acknowledgments/ (which does not yet exist).  I'll look into our options for also supporting redirect for the British spelling.

Reply:
https://data.desi.lbl.gov/doc/acknowledgments/ now does exist as the preferred link to use, and desihub/desidatadocs#60 will restore support for the older spelling for pre-deployed sites using the older URL.

Reply:
OK great. Marking as resolved.